templates/blocks/spreadsheettable.twig

```diff
@@ -0,0 +1,31 @@
+{% block spreadsheettable %}
+
+	<input class="spreadsheet-table-search" type="text" placeholder="Search by Name, Position or Country" />
+	<div class="spreadsheet-table">
+		{% for key, item in fields.spreadsheet_data %}
+
+			<div class="table-row">
```

Comment:
What is the reason for not using the html `<table>` element here to display a table?

Reply:
Because of handling responsive table, it is easier.